docs/_articles/en/release-information.md

```diff
@@ -5,6 +5,31 @@ lang: en
 
 ### To update the plugin, please follow the linked  [instructions](./en/getting-started/install/#upgrade-plug-in).
 
+## [v2.6.0](https://github.com/forcedotcom/sfdx-scanner/tree/v2.6.0) (02-17-2021)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.5.1...v2.6.0)
+
+### Release Summary
+
+* Support for SARIF JSON Format
+* Underlying PMD Engine is upgraded to 6.31.0 (from 6.30.0) 
+* RetireJS upgraded to v2.2.4
+
+**Closed issues:**
+
+- Generation of a code quality report in a JSON file that implements a subset of the Code Climate spec. [\#329](https://github.com/forcedotcom/sfdx-scanner/issues/329)
+- Support SARIF JSON Format [\#309](https://github.com/forcedotcom/sfdx-scanner/issues/309)
+- The Scanner should \(only\) use each engine default configs/rulesets [\#248](https://github.com/forcedotcom/sfdx-scanner/issues/248)
+
+**Merged pull requests:**
+
+- @W-8553897@ Include sarif run node for all engines [\#353](https://github.com/forcedotcom/sfdx-scanner/pull/353)
```

Comment:
Can we remove this, it's a fix for the larger story, but this was never released, so it might confuse users.

Reply:
ok